src/main/java/net/dries007/tfc/client/ClientEventHandler.java

```diff
@@ -838,7 +839,9 @@ public static void registerColorHandlerItems(RegisterColorHandlersEvent.Item eve
         }
 
         TFCItems.MOLDS.values().forEach(reg -> event.register(new ContainedFluidModel.Colors(), reg.get()));
-        event.register(new ContainedFluidModel.Colors(), TFCItems.WOODEN_BUCKET.get(), TFCItems.BELL_MOLD.get(), TFCItems.FIRE_INGOT_MOLD.get(), TFCItems.JUG.get(), TFCItems.SILICA_GLASS_BOTTLE.get(), TFCItems.HEMATITIC_GLASS_BOTTLE.get(), TFCItems.VOLCANIC_GLASS_BOTTLE.get(), TFCItems.OLIVINE_GLASS_BOTTLE.get());
+        TFCItems.GLAZED_JUGS.values().forEach(reg -> event.register(new ContainedFluidModel.Colors(), reg.get()));
+
+        event.register(new ContainedFluidModel.Colors(), TFCItems.WOODEN_BUCKET.get(), TFCItems.BELL_MOLD.get(), TFCItems.FIRE_INGOT_MOLD.get(), TFCItems.JUG.get(), TFCItems.SILICA_GLASS_BOTTLE.get(), TFCItems.HEMATITIC_GLASS_BOTTLE.get(), TFCItems.VOLCANIC_GLASS_BOTTLE.get(), TFCItems.OLIVINE_GLASS_BOTTLE.get(), TFCItems.RED_STEEL_BUCKET.get(), TFCItems.BLUE_STEEL_BUCKET.get());
```

Comment:
Are red/blue steel bucket models broken?

Reply:
They don't show the color of the liquid inside
![image](https://github.com/TerraFirmaCraft/TerraFirmaCraft/assets/78568080/b4904db2-b66f-468e-a101-64c8316fcdce)

This fixes that